src/blockdata/transaction.rs

```diff
@@ -416,37 +416,66 @@ impl Transaction {
         SigHash::from_engine(engine)
     }
 
-    /// Gets the "weight" of this transaction, as defined by BIP141. For transactions with an empty
-    /// witness, this is simply the consensus-serialized size times 4. For transactions with a
-    /// witness, this is the non-witness consensus-serialized size multiplied by 3 plus the
-    /// with-witness consensus-serialized size.
+    /// Returns the "weight" of this transaction, as defined by BIP141.
     #[inline]
+    #[deprecated(since = "0.29.0", note = "Please use `transaction::weight` instead.")]
     pub fn get_weight(&self) -> usize {
-        self.get_scaled_size(WITNESS_SCALE_FACTOR)
+        self.weight()
     }
 
-    /// Gets the regular byte-wise consensus-serialized size of this transaction.
+    /// Returns the "weight" of this transaction, as defined by BIP141.
+    ///
+    /// For transactions with an empty witness, this is simply the consensus-serialized size times
+    /// 4. For transactions with a witness, this is the non-witness consensus-serialized size
```

Comment:
nit: This will format like a numbered list (since the line starts with a number and dot).

Reply:
Lol. I guess we should change `4` to `four`.

Reply:
Ha, that's funny. Good catch @dr-orlovsky